content/documentation/pages/5-feature-guides/2-batch/6-composed-task.md

```diff
@@ -0,0 +1,643 @@
+
+Similarly if I wanted to pass a deployer property the format would remain the same except the the property type would `deployer`, for example we need to set the `kubernetes.limits.cpu` for `task-a`:
+
+```
+    deployer.my-composed-task.task-a.kubernetes.limits.cpu=1000m
+```
+
+Launching a composed task and setting both `app` and `deployer` properties would be done in the following way using the UI:
+
+1. Launch composed task as shown below by pressing the `play` button next to the composed task definition that needs to be launched:
+   ![Specify Which Composed Task to Launch](images/SCDF-composed-task-child-property-example-launch.png)
+1. Set the properties as follows in the `Parameters` text box:
+   ![Launch the Composed Task](images/SCDF-composed-task-child-property-launch-props.png)
+1. Now press the `Launch the task` button.
+
+## Passing Properties to Composed Task Runner
+
+There are 3 components that make up the property:
+
+- Property Type - This tells Spring Cloud Data Flow if the property is either a `deployment` or a `app` type.
+  - Deployment properties - are instructions to the deployer responsible for deploying the child task.
+  - App properties - are properties passed directly the child task.
+- Composed Task Application Name - Unlike when passing properties to a child task where we used the composed task definition name we will use the name of the composed task runner app.
+- Property Key - The key of the property that is to be set.
+
+Let's launch a composed task where we want to pass the following properties to the `Composed Task Runner`:
+
+- increment-instance-enabled - an `app` property that allows a single `Composed Task Runner` instance to be re-executed without changing the parameters.
+- kubernetes.limits.cpu - a `deployer` property that sets the Kubernetes cpu limit for the composed task runner
+  Launching a composed task and setting both `app` and `deployer` properties for the `Composed Task Runner` would be done in the following way using the UI:
+
+1. Launch composed task as shown below by pressing the `play` button next to the composed task definition that needs to be launched:
+   ![Specify Which Composed Task to Launch](images/SCDF-composed-task-child-property-example-launch.png)
+1. Set the properties as follows in the `Parameters` text box:
+   ![Launch the Composed Task](images/SCDF-composed-task-child-property-launch-ctr-props.png)
+1. Now press the `Launch the task` button.
+
+# Launching Composed Task using RESTFul API
+
+In this section we will provide an example on how to create and launch a composed-task.
+
+Let's say we want to create a composed task `my-composed-task` with the following composed task definition:
+
+```
+task-a && task-b
+```
+
+Using `curl` the command would look like:
+
+```shell script
+curl 'http://localhost:9393/tasks/definitions' -i -X POST -d 'name=my-composed-task&definition=task-a%20%26%26%20task-b'
```

Comment:
Maybe put the task definition in a file and then use the `@` option in curl to post that file because using `%20` to aovid spaces and encode `&&` makes it hard to author/read.

Reply:
how about `--data-urlencode "name=my-composed-task" --data-urlencode "definition=task-a && task-b"`  ?